app/src/main/java/com/github/fribourgsdp/radio/StringComparisons.kt

```diff
@@ -0,0 +1,98 @@
+package com.github.fribourgsdp.radio
+
+import java.text.Normalizer
+
+const val MAX_ALLOWED_ERRORS = 2
+
+class StringComparisons {
+    companion object{
+        private const val chars = "abcdefghijklmnopqrstuvwxyz1234567890"
+
+        /**
+        * Function that returns a number of points depending on the number of mistakes in the comparison of the two provided strings.
+        */
+        fun compareAndGetPoints(actual : String, expected : String, maxPts : Int = 100) : Int {
+            val oneErrorPenalty = 0.9
+            val twoErrorsPenalty = 0.8
```

Comment:
Don't you want to put these values in constant?

Reply:
Yeah it would be better.